pkg/rbuser/search.go

```diff
@@ -0,0 +1,97 @@
+package rbuser
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	ldap "gopkg.in/ldap.v2"
+)
+
+// SearchRB search redbrick ldap for a given filter and return first user that matches
+func SearchRB(l *ldap.Conn, filter string) (RBUser, error) {
```

Comment:
This would be better as part of `type RBLDAP struct`. It is not usual to pass connection like this.

Reply:
structure of `searchDCU` would need to be reconsidered for that